### DISCUSSION TITLE
Problem with get private indicator

### DISCUSSION THREAD

**Discussion by BlackPhoenixSlo** [Mathieu2301/TradingView-API#254] [Q&A]
Tradingview changed some metadata and even get private indicator doesn't work. 

`const TradingView = require('@mathieuc/tradingview');

/**
 * This example creates a chart with all user's private indicators
 */

process.argv[2] = "gkxr8eu7va2mgfdvcOMITED"
process.argv[3] = "v3:hR+9bFzBLoHRuCOMITED"
//const id = "7pxbwffocii753w2z5jjdq0xf3jb0kri";
//const certificate = "v3:BwppgofWBHNGegxr2nX7Za9neYzpMP7hzqaW71vx3xk=";
if (!process.argv[2]) throw Error('Please specify your \'sessionid\' cookie');
if (!process.argv[3]) throw Error('Please specify your \'signature\' cookie');

const client = new TradingView.Client({
  token: process.argv[2],
  signature: process.argv[3],
});

const chart = new client.Session.Chart();
chart.setMarket('BINANCE:BTCEUR', {
  timeframe: 'D',
});

TradingView.getPrivateIndicators(process.argv[2]).then((indicList) => {
  indicList.forEach(async (indic) => {

    try {
      const privateIndic = await indic.get();
      console.log('Loading indicator', indic.name, '...');
      const indicator = new chart.Study(privateIndic);
      indicator.onReady(() => {
        console.log('Indicator', indic.name, 'loaded !');
      console.log('Indicator', indic.id, 'id !');
      console.log('Indicator', indic.version, 'version !');
      console.log('Indicator', indic.type, 'type !');
      console.log('Indicator', indic.source, 'source !');
      });
    }
    catch (error) {
      console.error('Error:', error.message);
    }
   
  
    // indicator.onUpdate(() => {
    //   console.log('Plot values', indicator.periods);
    //   console.log('Strategy report', indicator.strategyReport);
    // });
  });
});
`

Here is result:

akabasej@Jakas-MacBook-Air TestPackagePotFullVersionPrivate % node _AllPrivateIndicators.js 
Credentials error: Wrong or expired sessionid/signature
^C
jakabasej@Jakas-MacBook-Air TestPackagePotFullVersionPrivate % node _AllPrivateIndicators.js
Credentials error: Wrong or expired sessionid/signature
^C
jakabasej@Jakas-MacBook-Air TestPackagePotFullVersionPrivate % node _AllPrivateIndicators.js
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"
Error: Inexistent or unsupported indicator: "undefined"

Bug is in .mist getPrivateindicator function

**Comment by hoangsoft90** [comment]
i got same error. how to fix it?
